.github/workflows/prettier.yml

```diff
@@ -16,7 +16,7 @@ jobs:
             - uses: bahmutov/npm-install@v1
 
             - name: Install
-              run: yarn
+              run: yarn --frozen-lockfile
```

Comment:
🥳

Reply:
turns out `bahmutov/npm-install@v1` already does it anyway :)